test/bioprinttest.c

```diff
@@ -11,8 +11,12 @@
 
 #include <stdio.h>
 #include <string.h>
-#include <openssl/opensslconf.h>
-#include OPENSSL_UNISTD
+#include <openssl/e_os2.h>
+#ifdef OPENSSL_SYS_WINDOWS
+# include <winsock.h>
```

Comment:
`write()`...  that's why I need `unistd.h`, which doesn't exist on Windows.  I had a look at `test/ssltest_old.c` to see how it was done there...